subsys/net/lib/lwm2m/lwm2m_engine.c

```diff
@@ -699,6 +784,71 @@ static u16_t atou16(u8_t *buf, u16_t buflen, u16_t *len)
 	return val;
 }
 
+static int atos32(u8_t *buf, u16_t buflen, s32_t *val)
```

Comment:
There's a strtol() in the minimal libc. Can that be used instead? (Should save a few hundred bytes.)

Reply:
Yes, good suggestion. I replace atos32() w/ strtol().